File: Math stuff/Project2/Math project1/VariableExpression.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Math_project1
{
    class VariableExpression : Expression
    {
        public override int evaluate(int x)
        {
            return x;
        }

        public override void printMe(int depth)
        {
            Console.WriteLine("-- x");
        }
    }
}

File: Math stuff/Project2/Expressions/src/com/william/expressions/VariableExpression.java
package com.william.expressions;

public class VariableExpression extends Expression {

	
	 public int evaluate(int x)
     {
		 
         return x;
     }
	 
	 
     public void printMe(int depth)
     {

    	 System.out.println("-- x");
     }
     
     public boolean equals(Object other) {
    	 return other instanceof VariableExpression;
     }

}
